test/pass/version-rocmversion-env.cmake

```diff
@@ -0,0 +1,18 @@
+################################################################################
+# Copyright (C) 2017 Advanced Micro Devices, Inc.
+################################################################################
+
+find_program(GIT NAMES git)
+
+set(ENV{ROCM_LIBPATCH_VERSION} "21001")
+file(MAKE_DIRECTORY ${TMP_DIR}/repo)
+test_exec(COMMAND ${GIT} init WORKING_DIRECTORY ${TMP_DIR}/repo)
+write_version_cmake(${TMP_DIR}/repo 1.0 "
+    test_expect_eq(\${PROJECT_VERSION_MAJOR} 1)
+    test_expect_eq(\${PROJECT_VERSION_MINOR} 0)
+    test_expect_eq(\${PROJECT_VERSION_PATCH} 0)
```

Comment:
There should be a `test_expect_eq(\${PROJECT_VERSION_TWEAK} \$ENV{ROCM_LIBPATCH_VERSION})`

Reply:
Updated as discussed.